### PR TITLE
Topic edits

### PR DESCRIPTION
**Pull request by philip-menzies**
docs\get_started\download\harbor_container_registry.md docs\get_started\download\software_licensing_portal\configure_entitlement_checks\configuring_local_flexnet_entitlement_server.md docs\get_started\download\software_licensing_portal\configure_entitlement_checks\entitlement_checks_scenarios.md docs\get_started\download\software_licensing_portal\configure_entitlement_checks\index.md docs\get_started\download\software_licensing_portal\index.md

### CODE REVIEW

File: docs/get_started/download/software_licensing_portal/configure_entitlement_checks/configuring_local_flexnet_entitlement_server.md [hunk 1]
```diff
@@ -115,98 +115,98 @@ A local Flexnet entitlement server can itself also be configured to function wit
...
+            3. Leave the **ID Type** as `ETHERNET`.
             
-            4. Under the **ID** field, enter the **hostids** value that you obtained.
+            4. Under the **ID** field, enter the **hostids** value that you obtained earlier.
         
-        4. Click **Save**. The device is created.
+        4. Click **Save**. The device is created.
 
-    2. Map your HCL HCL Digital Experience Cloud Native 9.5 entitlements to the local license server. Reference the table: [HCL Digital Experience Cloud Native 9.5 Tier 1 – 7 parts and HCL Flexnet License Server Feature Name](index.md) for details.
+    2. Map your  HCL Digital Experience Cloud Native 9.5 entitlements to the local license server. Refer to the following table: [HCL Digital Experience Cloud Native 9.5 Tier 1 – 7 parts and FlexNet License Server Feature Name](index.md) for details.
```
Comment (biancaluna) [inline docs/get_started/download/software_licensing_portal/configure_entitlement_checks/configuring_local_flexnet_entitlement_server.md:138]:
```suggestion
    2. Map your HCL Digital Experience Cloud Native 9.5 entitlements to the local license server. Refer to the following table: [HCL Digital Experience Cloud Native 9.5 Tier 1 – 7 parts and FlexNet License Server Feature Name](index.md) for details.
```

Comment (philip-menzies) [reply docs/get_started/download/software_licensing_portal/configure_entitlement_checks/configuring_local_flexnet_entitlement_server.md:138]: Yes

File: docs/get_started/download/software_licensing_portal/configure_entitlement_checks/entitlement_checks_scenarios.md [hunk 2]
```diff
@@ -1,92 +1,95 @@
...
+    Select your account and then click **Search**. Select **Digital Experience Cloud Native 9.5** from the list of your account entitlements. Results are displayed and are similar to the following example. 
 
     !!!note
-        The results may show an overage percentage amount in parentheses which is N/A for this informational dashboard report.
+        The results might show an overage percentage amount in parentheses which does not apply to this informational dashboard report.
 
         ![](../../software_licensing_portal/_img/select_account_entitlements.png)
 
-The Usage report will show:
+The Usage report will show the following information:
```
Comment (biancaluna) [inline docs/get_started/download/software_licensing_portal/configure_entitlement_checks/entitlement_checks_scenarios.md:80]:
```suggestion
The Usage report shows the following information:
```

Comment (philip-menzies) [reply docs/get_started/download/software_licensing_portal/configure_entitlement_checks/entitlement_checks_scenarios.md:80]: Yes

File: docs/get_started/download/software_licensing_portal/configure_entitlement_checks/index.md [hunk 3]
```diff
@@ -118,42 +118,42 @@ configuration:
 ```
 
 !!!reminder
-    These properties should be configured to your Helm chart BEFORE installing the environment or if making changes to the environment, before executing the DX 9.5 ContainerUpdate 207 or later Helm upgrade to your HCL Digital Experience Cloud Native 9.5 production or non-production deployment. For more information on the Helm configuration steps to manage DX 9.5 Container Update upgrades, reference this Help Center topic: [Upgrade the Helm deployment to the latest version](../../../../deployment/install/container/helm_deployment/update_helm_deployment.md).
+    These properties must be configured to your Helm chart **before** you install the environment. Or if you're changing the environment, configure the properties before you start the DX 9.5 ContainerUpdate 207 or later Helm upgrade to your HCL Digital Experience Cloud Native 9.5 production or non-production deployment. For more information about the Helm configuration steps to manage DX 9.5 Container Update upgrades, see [Upgrade the Helm deployment to the latest version](../../../../deployment/install/container/helm_deployment/update_helm_deployment.md).
```
Comment (biancaluna) [inline docs/get_started/download/software_licensing_portal/configure_entitlement_checks/index.md:121]:
```suggestion
    These properties must be configured to your Helm chart **before** you install the environment. If you're changing the environment, configure the properties before you start the DX 9.5 ContainerUpdate 207 or later Helm upgrade to your HCL Digital Experience Cloud Native 9.5 production or non-production deployment. For more information about the Helm configuration steps to manage DX 9.5 Container Update upgrades, see [Upgrade the Helm deployment to the latest version](../../../../deployment/install/container/helm_deployment/update_helm_deployment.md).
```

File: docs/get_started/download/software_licensing_portal/configure_entitlement_checks/index.md [hunk 4]
```diff
@@ -118,42 +118,42 @@ configuration:
...
 
-Use kubectl logs for the license manager pod. For example in a namespace dxns, execute the following command: 
+Use kubectl logs for the license manager pod. For example, in a namespace dxns, run the following command: 
 
 ```
 kubectl logs pod/<release-name>-license-manager-0 -n <namespace>
 ```
 
-See the HCL DX Cloud Native 9.5 entitlement check scenarios for success and error messages and how to manage in the [HCL Digital Experience Cloud Native 9.5 entitlement check scenarios](entitlement_checks_scenarios.md) topic. 
+See the HCL DX Cloud Native 9.5 entitlement check scenarios for success and error messages and how to manage them:  [HCL Digital Experience Cloud Native 9.5 entitlement check scenarios](entitlement_checks_scenarios.md).
```
Comment (biancaluna) [inline docs/get_started/download/software_licensing_portal/configure_entitlement_checks/index.md:134]:
```suggestion
See the HCL DX Cloud Native 9.5 entitlement check scenarios for success and error messages and how to manage them: [HCL Digital Experience Cloud Native 9.5 entitlement check scenarios](entitlement_checks_scenarios.md). 
```

Comment (philip-menzies) [reply docs/get_started/download/software_licensing_portal/configure_entitlement_checks/index.md:134]: Yes

**Review by philip-menzies — COMMENTED** [review]
Yes

**Pull request merged**